.github/workflows/check_links.yml

```diff
@@ -4,6 +4,8 @@ name: 'Check links'
 # yamllint disable-line rule:truthy
 on:
   workflow_dispatch:
+  schedule:
+    - cron: '37 11 * * 2'
```

Comment:
11:37's a bit... precise? But no matter 🙂

Reply:
I tend to pick random times, so as to reduce the chance of clashing with other workflows. The default people would go for is probably 0 minutes.

(That said, GitHub's scheduled times are not precise anyway: a trigger might be set for 11:00 and run at some random time a few minutes after.)